Makefile.am

```diff
@@ -134,6 +134,13 @@ test_run_tests_CFLAGS =
 else
 test_run_tests_CFLAGS = -Wno-long-long
 endif
+
+if SUNOS
+# Can't be turned into a CC_CHECK_CFLAGS in configure.ac, it makes compilers
```

Comment:
You mean this comment: https://github.com/libuv/libuv/blob/v1.x/configure.ac#L42?